data/json/monsters.json

```diff
@@ -2560,6 +2566,36 @@
     "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "BLEED", "GRABS", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_grappler",
+    "name": "grappler zombie",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"cyan_white",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":15,
+    "aggression":100,
+    "morale":100,
+    "speed":85,
+    "melee_skill":5,
+    "melee_dice":4,
+    "melee_dice_sides":5,
+    "melee_cut":0,
+    "dodge":2,
+    "armor_bash":0,
+    "armor_cut":0,
+    "vision_day":30,
+    "vision_night":5,
+    "death_drops": "default_zombie_death_drops",
+    "luminance":0,
+    "hp":110,
+    "death_function":["NORMAL"],
+    "special_attacks":[["BITE", 5], ["PULL_ENEMY", 20], ["GRAB_PULL", 2]],
```

Comment:
You should probably reorder the attacks - they are used in order in which they appear on the list. In this case it will bite before grabbing.